src/Jackett.Common/Indexers/AniLibria.cs

```diff
@@ -0,0 +1,189 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using AngleSharp.Dom;
+using AngleSharp.Html.Parser;
+using AngleSharp.Html.Dom;
+using Jackett.Common.Models;
+using Jackett.Common.Models.IndexerConfig;
+using Jackett.Common.Services.Interfaces;
+using Jackett.Common.Utils;
+using Jackett.Common.Utils.Clients;
+using Newtonsoft.Json.Linq;
+using NLog;
+using System.Text.RegularExpressions;
+
+namespace Jackett.Common.Indexers
+{
+    [ExcludeFromCodeCoverage]
+    public class AniLibria : BaseWebIndexer
+    {
+        public AniLibria(IIndexerConfigurationService configService, Utils.Clients.WebClient wc, Logger l, IProtectionService ps)
+            : base(id: "AniLibria",
+                   name: "AniLibria",
+                   description: "AniLibria is a Public torrent tracker for anime, voiced on russian by AniLibria team",
+                   link: "https://www.anilibria.tv/",
+                   caps: new TorznabCapabilities(),
+                   configService: configService,
+                   client: wc,
+                   logger: l,
+                   p: ps,
+                   configData: new ConfigurationData())
+        {
+            Encoding = Encoding.UTF8;
+            Language = "ru-ru";
+            Type = "public";
+
+            // Configure the category mappings
+            AddCategoryMapping(1, TorznabCatType.TVAnime, "Anime");
+        }
+
+        private Uri GetAbsoluteUrl(string relativeUrl) => new Uri(SiteLink + Regex.Replace(relativeUrl, @"\\/", "/").TrimStart('/'));
+
+        /// <summary>
+        /// https://www.anilibria.tv/public/search.php
+        /// </summary>
+        private Uri SearchUrl => GetAbsoluteUrl("/public/search.php");
```

Comment:
I think you can delete SearchUrl
In other trackers we have a private string attributes (you don't need to convert uri => string) => https://github.com/Jackett/Jackett/blob/621a47338dcd98ff0b2471b7df99f7995e10a4dd/src/Jackett.Common/Indexers/AwesomeHD.cs#L22